constellationdetector.py

```diff
@@ -36,7 +36,8 @@ def search_for_constellation(self, con, x, y, mags):
         matches = self.check_for_matches(cx, cy, x, y, test_scale)
         # If atleast half the stars match, draw the star
         lines = []
-        if matches >= 0.5*len(con.stars_x):
+        if matches >= 0.5*len(con.stars_x) and matches<len(con.stars_x):
```

Comment:
can this be changed to:
`if matches >= 0.5*len(con.stars_x) and matches<=len(con.stars_x):`
so that we can match every star in the template, just cant match MORE then every star?